stormlog/jax/tracker.py

```
@@ -182,16 +193,24 @@ def __init__(
-                        self._device_bytes_limit = int(stats["bytes_limit"])
-                except Exception:
-                    pass
+            else:
+                self._device, self.device_index = resolve_jax_device(device_index)
+            capability = get_device_memory_capability(self._device)
+            self._device_memory_available = bool(capability["memory_stats_available"])
+            self._device_memory_unavailable_reason = capability["memory_stats_error"]
+            stats = capability["memory_stats"]
+            if self._device_memory_available and "bytes_limit" in stats:
+                self._device_bytes_limit = int(stats["bytes_limit"])
         except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)